test/data/acceptance.json

```diff
@@ -1,7 +1,7 @@
 {
   "html": [
-    "<!DOCTYPE html>\n<html>\n    <head>\n        <title>Unit 1, Project 1</title>\n    </head>\n    <body>\n        <h1>I love ScriptEd!</h1>\n        <img src=\"http://i.imgur.com/J5ji6zX.png\">\n        <p>This is the ScriptEd logo</p>\n        <a href=\"www.scripted.org\">This is a link to ScriptEd's website </a>\n        \n        <img src=\"http://i.imgur.com/EwUGnTi.png\">\n        <p>In ScriptEd we learn the HTML, CSS and JavaScript</p>\n        <a href=\"https://github.com/ScriptEdcurriculum/curriculum2016\">This is a link to ScriptEd's curriculum </a>\n    </body>\n</html>\n",
-    "<!DOCTYPE html>\n<html>\n    <head>\n        <title>Unit 1, Project 1</title>\n    </head>\n    <body>\n        <h1>I love ScriptEd!</h1>\n        <img src=\"http://i.imgur.com/J5ji6zX.png\">\n        <p>This is the ScriptEd logo</p>\n        <a href=\"www.scripted.org\">This is a link to ScriptEd's website </a>\n        \n        <img src=\"http://i.imgur.com/EwUGnTi.png\">\n        <p>In ScriptEd we learn the HTML, CSS and JavaScript</p>\n        <a href=\"https://github.com/ScriptEdcurriculum/curriculum2016\">This is a link to ScriptEd's curriculum </a>\n    </body>\n</html>\n",
+    "<!DOCTYPE html>\n<html>\n    <head>\n        <title>Unit 1, Project 1</title>\n    </head>\n    <body>\n        <h1>I love ScriptEd!</h1>\n        <img src=\"http://i.imgur.com/J5ji6zX.png\">\n        <p>This is the ScriptEd logo</p>\n        <a href=\"https://www.scripted.org\">This is a link to ScriptEd's website </a>\n        \n        <img src=\"http://i.imgur.com/EwUGnTi.png\">\n        <p>In ScriptEd we learn the HTML, CSS and JavaScript</p>\n        <a href=\"https://github.com/ScriptEdcurriculum/curriculum2016\">This is a link to ScriptEd's curriculum </a>\n    </body>\n</html>\n",
```

Comment:
@outoftime Finally figured out that it was this acceptance test that was failing. Added the test name when we run `testValidatorAcceptance()` so we catch it quicker in the future.